egs/librispeech/ASR/distillation_with_hubert.sh

```diff
@@ -0,0 +1,95 @@
+stage=$1
```

Comment:
What if no `stage` is provided by the user?
Could you add any usage information to this file?

Reply:
A example usage is added.
I hope users could run this demo script stage by stage before they know what each stage does.
They they can modify this script as they want.